modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/paths/PRMMDFRequest.path

```diff
@@ -110,6 +110,16 @@
 	<td>//div[contains(@class,'edit-button-mdf')]/div</td>
 	<td></td>
 </tr>
+<tr>
+	<td>REMOVE_VALUE_BUDGET_BUTTON</td>
+	<td>//button[contains(@class,'btn-monospaced')]/*[name()='svg'][contains(@class,'lexicon-icon lexicon-icon-hr')]</td>
+	<td></td>
+</tr>
```

Comment:
Hi @Italo-gabriel, this path isn't necessary. You can use `Icon#ANY`.

Reply:
Done !